src/runtime/crypto/__tests__/produce-node-hmac.mjs

```diff
@@ -0,0 +1,33 @@
+import crypto from 'crypto';
+
+function random({min = 0, max = 1, floor = true} = {}) {
+  let value = Math.random() * (max - min) + min;
+  if (floor) {
+    value = Math.floor(value);
+  }
+  return value;
+}
```

Comment:
Since this is duplicated in `produce-node-base64.mjs`, opportunity to put in a shared file and imported into each script?